docs/reference/migration/migrate_2_1.asciidoc

```diff
@@ -25,13 +25,3 @@ GET /my_index/_search?scroll=2m
 Scroll requests sorted by `_doc` have been optimized to more efficiently resume
 from where the previous request stopped, so this will have the same performance
 characteristics as the former `scan` search type.
-
-=== Mapping changes
```

Comment:
Do we need this here ?  https://github.com/elastic/elasticsearch/blob/master/docs/reference/migration/migrate_2_0/mapping.asciidoc